modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -22,9 +22,14 @@ const getContentFromBody = (editor: Editor, args: GetContentArgs, body: HTMLElem
   if (args.format === 'raw') {
     content = Tools.trim(TrimHtml.trimExternal(editor.serializer, body.innerHTML));
   } else if (args.format === 'text') {
-    // return empty string for text format when editor is empty to avoid bogus elements being returned in content
-    content = Zwsp.trim(body.innerText);
-    content = content === '\n' ? '' : content;
+    content = body.innerHTML.replace(/<br data-mce-bogus="1">/g, '\uFEFF');
+    const parseBody = document.createElement('body');
+    parseBody.setAttribute('style', 'position:fixed;left:400%;top:0px');
```

Comment:
As discussed on slack, this won't work with stricter CSP's and may still show in some edge cases... please use the normal method of a negative position instead.